docs/dev/core/internal/payload.md

```diff
@@ -157,6 +157,27 @@ To get around this, on application startup, any queued events are immediately co
 These "startup-triggered pings" are likely to have a very short duration, as recorded in `ping_info.start_time` and `ping_info.end_time` (see [the `ping_info` section](../../../user/pings/index.md#The-ping_info-section)).
 The maximum timestamp of the events in these pings are quite likely to exceed the duration of the ping, but this is to be expected.
 
+### Custom Distribution
+
+A [Custom distribution](../../../user/metrics/custom_distribution.md) is represented as an object with the following fields.
+
+| Field name | Type | Description |
+|---|---|---|
+| `sum` | Integer | The sum of all recorded values. |
+| `values` | Map&lt;String, Integer&gt; | The values in each bucket. The key is the minimum value for the range of that bucket. | 
```

Comment:
Can we also mention that this is reporting all buckets [0, max)? A small note somewhere about the rationale behind this would be very useful for our future selves, IMHO.!